superset-frontend/src/common/components/Modal.tsx

```diff
@@ -78,11 +78,6 @@ const StyledModal = styled(BaseModal)`
       margin-left: 8px;
     }
   }
-
-  // styling for Tabs component
```

Comment:
Curious if/how this is relevant to the PR... looks like it would be a visual change... is it something @kgabryje is aware of?

Reply:
It looks like a rebasing mistake to me. Adam, can you please bring it back?